src/modules/admissions/views/AdmissionStatistics.tsx

```diff
@@ -0,0 +1,106 @@
+import { useQuery } from '@apollo/client'
+import {
+  Card,
+  Center,
+  Container,
+  createStyles,
+  SimpleGrid,
+  Stack,
+  Text,
+  Title,
+} from '@mantine/core'
+import { Breadcrumbs } from 'components/Breadcrumbs'
+import { CardTable } from 'components/CardTable'
+import { FullPageError } from 'components/FullPageComponents'
+import { FullContentLoader } from 'components/Loading'
+import { useIsMobile } from 'util/hooks'
+import { INTERVIEW_STATISTICS_QUERY } from '../queries'
+import { InterviewStatisticsReturns } from '../types.graphql'
+
+const AdmissionStatistics: React.FC = () => {
+  const { classes } = useStyles()
+  const isMobile = useIsMobile()
+  const { data, loading, error } = useQuery<InterviewStatisticsReturns>(
+    INTERVIEW_STATISTICS_QUERY
+  )
+
+  if (error) return <FullPageError />
+
+  if (loading || !data) return <FullContentLoader />
+
+  const {
+    interviewStatistics: {
+      totalApplicants,
+      totalAvailableInterviews,
+      totalBookedInterviews,
+      totalFinishedInterviews,
+      userInterviewCounts,
+    },
+  } = data
+
+  return (
+    <Stack>
+      <Breadcrumbs
+        items={[
+          { label: 'Hjem', path: '/dashboard' },
+          { label: 'Orvik', path: '/admissions' },
+          { label: 'Statistikk', path: '' },
+        ]}
+      />
+      <Title>Statistikk</Title>
+      <SimpleGrid cols={isMobile ? 2 : 4}>
```

Comment:
we should probably decide whether to use the included breakpoints function for columns or use it like you do here.